docs/configuration.md

```diff
@@ -194,6 +194,27 @@ The names of the dimensions (in order) that will appear *pinned* by default on t
 
 Data cube introspection strategy.
 
+**refreshRule**
+
+Refresh rule defining how is the information about latest data in a data source obtained.
```

Comment:
how the information is obtained

Reply:
it is described in **Refresh rules** section below